content/blog/2020/10/Camel36-Whatsnew/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Apache Camel 3.6 What's New"
+date: 2020-10-21
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Details of what we have done in the Camel 3.6 release.
+---
+
+Apache Camel 3.6 has just been released.
+
+This is a non-LTS release which means we will not provide patch releases, but use the release _as-is_.
+The next planned LTS release is 3.7 scheduled towards end of the year.
+
+
+## So what's in this release?
+
+This release introduces new set of features and noticeable improvements that will we cover in this blog post.
+
+
+### Spring Boot
+
+We have upgraded to latest release at this time which is Spring Boot 2.3.4.
+Support for Spring Boot 2.4 is planned for Camel 3.7.
+
+
+### Optimizations
+
+To speedup startup we switched to a new uuid generator. The old (classic) generator was inherited from Apache ActiveMQ which needed to ensure its ids were unique in a network of brokers, and therefore to ensure this the generator was using the hostname as prefix in the id. This required on startup to do a network access to obtain this information which costs a little time. Also depending on networks this can be more restrictive and delay the startup. The new generator is a pure in-memory fast generator that was used by Camel K and Camel Quarkus.
```

Comment:
```suggestion
To speed up the startup we switched to a new UUID generator. The old (classic) generator was inherited from Apache ActiveMQ which needed to ensure its ids were unique in a network of brokers, and therefore to ensure this the generator was using the hostname as the prefix in the id. This required on startup to do network access to obtain this information which costs a little time. Also depending on networks this can be more restrictive and delay the startup. The new generator is a pure in-memory fast generator that was used by Camel K and Camel Quarkus.
```